mappings/net/minecraft/structure/pool/StructurePoolElement.mapping

```diff
@@ -30,3 +30,30 @@ CLASS net/minecraft/class_3784 net/minecraft/structure/pool/StructurePoolElement
 		ARG 3 rotation
 	METHOD method_16757 getType ()Lnet/minecraft/class_3816;
 	METHOD method_19308 getGroundLevelDelta ()I
+	METHOD method_30421 newFeaturePoolElement (Lnet/minecraft/class_2975;)Ljava/util/function/Function;
```

Comment:
Do we use names like `ofFeature` `ofLegacySingle` `ofProcessedLegacySingle`

Reply:
I think so. I'll update my PR to use those names then. Glad their shorter 😄